tcms/bugs/tests/test_models.py

```diff
@@ -47,3 +48,11 @@ def test_no_notification_if_assignee_not_set(self, send_mail):
         BugFactory(assignee=None)
 
         self.assertFalse(send_mail.called)
+
+
+class TestBugTagPermissions(TestCase):
+    def test_add_bugtag(self):
+        self.assertTrue(Permission.objects.filter(codename='add_bugtag'))
```

Comment:
this can be 
```python
self.assertTrue(Permission.objects.filter(codename='add_bugtag').exists())
``` 
that way is even more clear that we are testing for the existence of such objects
- https://docs.djangoproject.com/en/3.0/ref/models/querysets/#exists